### DISCUSSION TITLE
Displaying example notebook

### DISCUSSION THREAD

**Discussion by aiqc** [KarchinLab/open-cravat#256] [Q&A]
The link to this file on the [Resources page](https://opencravat.org/resources.html) displays in raw JSON:

https://opencravat.org/oc_jupyter_python.ipynb

GitHub is capable of displaying ipynb files, so you could just link to the source file in the repo instead

**Comment by RachelKarchin** [comment]
Thanks for catching this! We now link to the source file in the repo for this notebook.
